GVRf/Framework/jni/gl/gl_texture.h

```diff
@@ -32,7 +32,7 @@ class GLTexture {
         glBindTexture(target, id_);
         glTexParameteri(target, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
         glTexParameteri(target, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
-        glTexParameteri(target, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
+        glTexParameteri(target, GL_TEXTURE_MIN_FILTER, GL_LINEAR_MIPMAP_LINEAR);
```

Comment:
I don't know if this was in the last check-in, if it was, i missed it.  linear_mipmap_linear is generally a lot more expensive than linear.  i don't feel comfortable defaulting to that.  but we should expose a mechanism for changing it.  Actually SIDIA also has a request for adjusting default texture parameters.